packages/trpc/server/routers/viewer/eventTypes/getEventTypesFromGroup.handler.ts

```diff
@@ -26,14 +31,57 @@ export const getEventTypesFromGroup = async ({ ctx, input }: GetByViewerOptions)
 
   const userProfile = ctx.user.profile;
   const { group, limit, cursor, filters } = input;
-  const { teamId, parentId } = group;
+  const { teamId } = group;
 
   const isFilterSet = (filters && hasFilter(filters)) || !!teamId;
   const isUpIdInFilter = filters?.upIds?.includes(userProfile.upId);
 
   const shouldListUserEvents =
     !isFilterSet || isUpIdInFilter || (isFilterSet && filters?.upIds && !isUpIdInFilter);
 
+  const eventTypes: MappedEventType[] = [];
+  let currentCursor = cursor;
+  let nextCursor: typeof cursor | undefined = undefined;
+  let isFetchingForFirstTime = true;
```

Comment:
isFetchingForFirstTime is required to prevent infinite while loop when user doesn't have any event types